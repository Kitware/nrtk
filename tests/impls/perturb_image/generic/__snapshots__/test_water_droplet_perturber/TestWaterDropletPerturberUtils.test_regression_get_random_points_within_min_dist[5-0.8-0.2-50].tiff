version https://git-lfs.github.com/spec/v1
oid sha256:cc063b639318352dbbaf4ad1f31901b2538053f8a4847c6e5a06c21bfebb8650
size 174
