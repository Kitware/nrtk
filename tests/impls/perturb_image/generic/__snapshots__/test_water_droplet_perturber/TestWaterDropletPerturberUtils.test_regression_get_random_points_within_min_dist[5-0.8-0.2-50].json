{"__type__": "ndarray", "dtype": "float64", "shape": [5, 2], "data": [[0.04298230670937811, 0.521017226391265], [0.3065127749388594, 0.06778379436644118], [0.6274628092148877, 0.32244577781883116], [0.76706260533518, 0.6037573724872068], [0.5488246032235187, 0.00769001633093156]]}
